### ISSUE TITLE
DotNet Proxy not working for windows authenticated service

### ISSUE THREAD

**Issue by haripraveen** [Esri/resource-proxy#382]
We are on ArcServer 10.3.1 and when we try to use the proxy on a windows authenticating service, it is not authenticating the user

**Comment by jgravois** [comment]
are you reporting that the credentials you're storing in your proxy.config are not being honored or that the actual end users credentials aren't being passed through?

**Comment by HCammus** [comment]
Hi, I tried to use the proxy on a windows authenticating service but username, password and domain passed empty.

**Comment by odoe** [comment]
I had someone contact me with a similar issue.

It looks like sending a URL via DotNet proxy with `https://<username>:<password>@hostname.com` doesn't pass the username and password along with the request.

This does work using the PHP proxy, so not sure what the issue may be.

**Comment by pfoppe** [comment]
Hi @haripraveen  

Our agency wanted to use the .Net Resource Proxy to authenticate to services that were setup and protected with [Microsoft Integrated Windows Authentication (IWA) using the MS Negotiate security support provider (Kerberos and/or NTLM)](https://msdn.microsoft.com/en-us/library/windows/desktop/aa378748(v=vs.85).aspx).  Our implementation approach was to setup an IIS Application Pool and configured that app pool to use a 'headless service account' (an account in our domain that its purpose was to access the secured services).  The .net proxy runs in this app-pool.  We wanted this approach to better protect the service account credentials/password (specifically get the clear-text out of the proxy.config file).  

The resource proxy code did not support this approach, so we forked the repo to include this possibility and our [pull request was merged in May 2017](https://github.com/Esri/resource-proxy/pull/309).  As documented in the [DotNet Proxy-Config Readme](https://github.com/Esri/resource-proxy/blob/master/README.md#proxy-configuration-settings)... Add a boolean **useAppPoolIdentity** parameter to the  to the ServerUrl entry (to the secured service).  

Let me know how it goes if you give it a try!
